native/ax/MpegDeMux-3/CDemuxer.cpp

```diff
@@ -911,15 +911,18 @@ HRESULT CDemuxer::SetMpeg2VideoMediaType(CMediaType *cmt, MPEG_VIDEO *pMpegVideo
     cmt->majortype = MEDIATYPE_Video;
     cmt->subtype = MEDIASUBTYPE_MPEG2_VIDEO;
 
+    int header_padded_length = pMpegVideo->actual_header_length + (4 - (pMpegVideo->actual_header_length % 4)) % 4; // pad length to DWORD boundary
```

Comment:
How about you put an ALIGN macros in the file like this and use that instead?

#define ALIGN(A, B) (((A) + (B)-1) & ~((B)-1))

Then it'd be:
int header_padded_length = ALIGN(pMpegVideo->actual_header_length, 4);